package.json

```diff
@@ -5,6 +5,7 @@
     "exchanges/*"
   ],
   "scripts": {
+    "install-frozen": "yarn install --frozen-lockfile",
```

Comment:
Accidental commit? Doesn't seem to be in use anywhere, and if it was in Circle we'd probably just add it inline

Reply:
Yeah This one was for forcing codesandbox to use a frozen lockfile but no dice either way

```suggestion
```